.travis.yml

```diff
@@ -32,7 +32,7 @@ before_install:
     chmod +x docker-compose
     sudo mv docker-compose /usr/local/bin
 
-install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q -T 2
+install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q -T 2 -pl '!presto-docs,!presto-server-rpm'
```

Comment:
I don't see it is going to be built in any of jobs specified in the section `script`. In the `script` maven is called with a goal `test` and here it is called with `install` which means that even modules `presto-docs` and `presto-server-rpm` are included the `rpm` file can be not built.

Maybe create a separate job (entry in `script`) which will make a full `mvn install`?